docs/integrations/github_actions/aws_oidc.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Authenticate to AWS with OIDC
+sidebar_label: AWS + OIDC
+---
+
+# Authenticate to AWS with OIDC
+
+If you run Steampipe in a [GitHub Action](https://steampipe.io/docs/integrations/github_action) you can use GitHub Actions Secrets to store the credentials that Steampipe uses to access AWS, Azure, GCP, or another cloud API. But what if you don't want to persist credentials there? An alternative is to use OpenID Connect (OIDC) to enable an Actions workflow that acquires temporary credentials on demand.
+
+The example shown in this post uses the OIDC method in a workflow that:
+
+1. Installs Steampipe (along with a cloud-specific plugin and compliance mod).
+2. Runs the Steampipe Compliance Mod for AWS and saves the output in the repository.
+
+## What is OIDC?
+
+[OpenID Connect 1.0](https://openid.net/specs/openid-connect-core-1_0.html) is an identity layer on top of the [OAuth 2.0 protocol](https://www.rfc-editor.org/rfc/rfc6749). It enables clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
+
+## Define the workflow
+
+Create a GitHub Actions workflow in your repository, as `.github/workflows/steampipe.yml`, based on [this example](https://github.com/turbot/steampipe-samples/blob/main/all/github-actions-oidc/aws/steampipe-sample-aws-workflow.yml). Let's review the key elements of the workflow.
```

Comment:
`We first must create the GitHub Actions workflow file in your repository. For this [example](https://github.com/turbot/steampipe-samples/blob/main/all/github-actions-oidc/aws/steampipe-sample-aws-workflow.yml) we will use the filename `.github/workflows/steampipe.yml``

This gives the reader the context of what they're about to do.